generators/docker-compose/templates/_docker-compose.yml

```diff
@@ -61,10 +61,12 @@ services:
         extends:
             file: jhipster-console.yml
             service: jhipster-console
+    <%_ if (composeApplicationType === 'microservice') { _%>
```

Comment:
@PierreBesson - This is defined in docker-compose (/docker-compose/prompt.js) - prompt to identify whether micro or monoliths. Gateway is not required, anyway it comes under microservices and the 'composeApplicationType' holds either microservice or monolith.